tensorflow/python/kernel_tests/decode_image_op_test.py

```diff
@@ -33,6 +33,17 @@
 
 class DecodeImageOpTest(test.TestCase):
 
+  def testBmp(self):
+    # Read a real bmp and verify shape
+    path = os.path.join(prefix_path, "bmp", "testdata", "lena.bmp")
```

Comment:
NotFoundError: tensorflow/core/lib/bmp/testdata/lena.bmp
	 [[Node: ReadFile = ReadFile[_device="/job:localhost/replica:0/task:0/cpu:0"](ReadFile/filename)]]


It looks like you need to add this file as testdata in the BUILD rule.